src/DragSelect.vue

```diff
@@ -47,12 +66,13 @@
       },
       selectionBoxStyling () {
         // Only set styling when necessary
-        if (!this.mouseDown || !this.startPoint || !this.endPoint) return {}
+        if (!this.mouseDown || !this.startPoint || !this.endPoint) return {background: this.color}
```

Comment:
Please don't exceed 80 characters per line and make sure to use [standard.js](https://standardjs.com/) code style.

Reply:
Moved the return statement to its own line (bracketed) and added spacing around the object statement